src/test/java/step2/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,47 @@
+package step2;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringAddCalculatorTest {
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
+        int result = StringAdderCalculator.splitAndSum(null);
+        assertThat(result).isEqualTo(0);
+
+        result = StringAdderCalculator.splitAndSum("");
+        assertThat(result).isEqualTo(0);
+    }
```

Comment:
예측되는 값이 동일하네요.
`null` 값은 다음과 같은 방법으로 검증할 수 있어요!

```suggestion
    @ParameterizedTest
    @CsvSource(value = {"null", "''"}, nullValues = {"null"})
    public void splitAndSum_null_또는_빈문자(String source) {
        int result = StringAdderCalculator.splitAndSum(source);
        assertThat(result).isEqualTo(0);
    }
```